coredump_uploader/__init__.py

```diff
@@ -475,6 +476,7 @@ def upload(self, path_to_core):
             stdin=subprocess.PIPE,
         )
         elfutils_version, err = process.communicate()
+        elfutils_version = elfutils_version.decode("utf-8")
```

Comment:
Since all these `.decode()` calls are for collecting context I think they should not be fatal if they fail.  So it's probably better to use one of the more lenient error handlers instead of the default `strict` one which will raise an exception.
Maybe `.decode("utf-8", errors="replace")` is a better choice for all of these?

Reply:
Yes, good. point

Reply:
Actually, this bit needs to be py2/3 safe.  adding the decode in 2.7 will turn the result into unicode, not what we want.
Will fix.